src/evaluate/__init__.py

```diff
@@ -26,11 +26,54 @@
 
 del version
 
-from .evaluator import Evaluator
+from typing import Dict, List
+
+from datasets import load_metric
+from datasets.metric import Metric
+from transformers.pipelines import SUPPORTED_TASKS as SUPPORTED_PIPELINE_TASKS
+from transformers.pipelines import TASK_ALIASES
+from transformers.pipelines import check_task as check_pipeline_task
+
+from .evaluator import Evaluator, TextClassificationEvaluator
+from .hub import push_to_hub
 from .info import EvaluationModuleInfo
 from .inspect import inspect_evaluation_module, list_evaluation_modules
 from .loading import load
 from .module import EvaluationModule
 from .saving import save
 from .utils import *
 from .utils import gradio, logging
+
+
```

Comment:
I would put these helper variables and functions in the evaluator.py module to keep the __init__.py lean.